railties/lib/rails/generators/rails/app/templates/config/initializers/new_framework_defaults_6_0.rb.tt

```diff
@@ -22,3 +22,11 @@
 # Send Active Storage analysis and purge jobs to dedicated queues.
 # Rails.application.config.active_storage.queues.analysis = :active_storage_analysis
 # Rails.application.config.active_storage.queues.purge    = :active_storage_purge
+
+# Use ActionMailer::MailDeliveryJob for sending parameterized and normal mail.
+#
+# This setting is not backwards compatible with earlier Rails versions.
+# If you send mail in the background, job workers need to have a copy of
+# MailDeliveryJob to ensure all delivery jobs are processed properly.
+# Make sure your entire app is migrated and stable on 6.0 before using this setting.
+Rails.application.config.action_mailer.delivery_job = "ActionMailer::MailDeliveryJob"
```

Comment:
It’s not obvious reading this as a user to understand why I’d want to switch. This option is also different from the other ps in that we intend to remove the old job in Rails 6.1, no?

It should also be commented out by default.

Reply:
I've commented it out, and I've added that the standard delivery job will be removed in 6.1. Is that and the deprecation warning when mail jobs are performed is enough?